### PR TITLE
Add descriptive logs

### PR DESCRIPTION
**Pull request by KunduzB**
## What does this PR do?

- Add more descriptive logs for private data

### CODE REVIEW

**Comment by boseok-bc** [comment]
@KunduzB could you add the examples?

File: cryptofeed/exchanges/bitmex.py [hunk 1]
```diff
@@ -790,9 +790,9 @@ async def _balance(self, msg: dict, timestamp: float):
         }
         """
 
-        LOG.info("Handle margin msg")
         for data in msg['data']:
             parsed_data = self.parse_margin(data)
+            LOG.info("Handle margin msg:                %s", list(parsed_data.items())[:3])
```
Comment (boseok-bc) [inline cryptofeed/exchanges/bitmex.py]: The spaces are intended?

File: cryptofeed/exchanges/bitmex.py [hunk 2]
```diff
@@ -636,10 +636,10 @@ async def _order(self, msg: dict, timestamp: float):
         }
         """
 
-        LOG.info("Handling order msg")
         for data in msg['data']:
             if data.get('ordStatus'):
                 new_info = self.parse_order(data)
+                LOG.info("Handling %s order msg: %s", new_info.get('symbol'), list(new_info.items())[:4])
```
Comment (boseok-bc) [inline cryptofeed/exchanges/bitmex.py]:
```suggestion
                LOG.info(f"Handling {new_info.get('symbol')} order msg: {list(new_info.items())[:4]}")
```

File: cryptofeed/exchanges/bitmex.py [hunk 3]
```diff
@@ -737,9 +737,9 @@ async def _position(self, msg: dict, timestamp: float):
         }
         """
 
-        LOG.info("Handle position msg")
         for data in msg['data']:
             parsed_data = self.parse_position(data)
+            LOG.info("Handle %s position msg: %s", parsed_data.get('symbol'), list(parsed_data.items())[:3])
```
Comment (boseok-bc) [inline cryptofeed/exchanges/bitmex.py]: ditto. f-string :)

File: cryptofeed/exchanges/bitmex.py (hunk 1 above)
Comment (boseok-bc) [reply cryptofeed/exchanges/bitmex.py]: ditto f-string

**Review by boseok-bc — COMMENTED** [review]
please use f-string

**Comment by KunduzB** [comment]
@boseok-bc  Example logs: https://www.notion.so/blockcrafterskor/Nile-f7e6faba277c4c42bb5fdc221a4094b9#5f4b280f8ec04bc9a2b8c700e41c53e0

**Review by boseok-bc — APPROVED** [review]
Good 👍

**Pull request merged**